numpy/random/mtrand/mtrand.pyx

```diff
@@ -1270,7 +1270,15 @@ cdef class RandomState:
         If ``high`` < ``low``, the results are officially undefined
         and may eventually raise an error, i.e. do not rely on this
         function to behave when passed arguments satisfying that
-        inequality condition.
+        inequality condition. Also, if the ``size`` is large enough
+        ( 10^6 or more ), the ``high`` limit may be included in the
+        returned array of floats due to floating-point rounding in the
+        equation ``low + (high-low) * random_sample()``. For example:
+
+        >>> x = np.float32(5*0.99999999)
+        >>> x
+        5.0
+
```

Comment:
It is not dependent on size, only on rounding. So something like "... inequality condition. The ``high`` limit may ...". The example is good.